<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work Area - Boulder Media - Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="1" rowspan="2">
       <widget class="QTextEdit" name="textEdit_writeDescription"/>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pushButton_saveWorkArea">
        <property name="text">
         <string>Save Work Area</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLineEdit" name="lineEdit_loadWorkArea"/>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_loadWorkArea">
          <property name="text">
           <string>Load Work Area</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_storeWorkArea">
          <property name="text">
           <string>Store Work Area</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLineEdit" name="lineEdit_storeWorkArea"/>
        </item>
       </layout>
      </item>
      <item row="8" column="0">
       <widget class="QPushButton" name="pushButton_restoreWorkArea">
        <property name="text">
         <string>Restore Work Area</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QListWidget" name="listWidget_versions"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Structure</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QTextEdit" name="textEdit_DescriptionFromVersion">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Description From Version</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Restore Version</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" rowspan="2">
       <widget class="QTreeView" name="treeView_folders"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>526</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
